packages/create-gatsby/src/init-starter.ts

```diff
@@ -16,25 +17,21 @@ const kebabify = (str: string): string =>
     .replace(/[^a-zA-Z]+/g, `-`)
     .toLowerCase()
 
-export const getPackageManager = (): PackageManager =>
-  getConfigStore().get(packageManagerConfigKey)
+export const packageManager = (npmConfigUserAgent?: string): PackageManager => {
```

Comment:
I don't like the change in the name of this function, as it's not clear now what it does. It's much better if function names are verb phrases.